tests/functional/test_dispatcher.c

```diff
@@ -26,6 +26,9 @@ usage(void)
 int
 test_dispatcher(int argc, char *argv[])
 {
+   // Info messages go to stdout
```

Comment:
(Sorry, very minor nit.)

You could remove this comment entirely as it's implied by the interface and the call itself.

Or, expand it to state both parts: `// Info messages go to stdout, and error messages to stderr`

The way it is right now it's high-lighting the former, making the reader wonder what's so special about info-msgs.

Reply:
Sure thing, fixed in latest.